r/tests/testthat/test-dplyr-mutate.R

```diff
@@ -652,3 +672,41 @@ test_that("Can use across() within transmute()", {
     example_data
   )
 })
+
+test_that("across() does not select grouping variables within mutate()", {
+  compare_dplyr_binding(
+    .input %>%
+      group_by(chr) %>%
+      mutate(across(everything(), round)) %>%
+      collect(),
+    example_data %>%
+      select(int, dbl, chr)
```

Comment:
nit: it's more skimmable when the `tbl` parameter to `compare_dplyr_binding()` is just a table with no modification. Please could you move the `select()` to the `expr` parameter code instead?